### ISSUE TITLE
React fails to recognize the prop named `fetchPriority`

### ISSUE THREAD

**Issue by Fun117** [shuding/nextra#2869]
By changing `fetchPriority` to `fetchpriority` in the `getDynamicProps` function located in `node_modules/next/dist/client/image-component.js`, the error was resolved. Is this issue on the Next.js side?

## Error Code
```bash
Warning: React does not recognize the `fetchPriority` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `fetchpriority` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    at img
    at eval (webpack-internal:///./node_modules/next/dist/client/image-component.js:136:11)
    at eval (webpack-internal:///./node_modules/next/dist/client/image-component.js:245:47)
    at div
    at div
    at div
    at div
    at MDXContent (webpack-internal:///./pages/index.mdx:1288:114)
    at MDXProvider (file:///Users/fun117/_dev/github/selcold/sglid/website/node_modules/@mdx-js/react/lib/index.js:90:30)
    at div
    at Body (webpack-internal:///./node_modules/nextra-theme-docs/dist/index.js:2918:3)
    at ActiveAnchorProvider (webpack-internal:///./node_modules/nextra-theme-docs/dist/index.js:156:3)
    at div
    at div
    at InnerLayout (webpack-internal:///./node_modules/nextra-theme-docs/dist/index.js:2969:3)
    at m (/Users/fun117/_dev/github/selcold/sglid/website/node_modules/next-themes/dist/index.js:1:310)
    at exports.ThemeProvider (/Users/fun117/_dev/github/selcold/sglid/website/node_modules/next-themes/dist/index.js:1:3611)
    at ConfigProvider (webpack-internal:///./node_modules/nextra-theme-docs/dist/index.js:262:3)
    at Layout (webpack-internal:///./node_modules/nextra-theme-docs/dist/index.js:3088:5)
    at Nextra (webpack-internal:///./node_modules/nextra/dist/layout.js:17:3)
    at App (webpack-internal:///./pages/_app.mdx:25:34)
    at MDXContent
    at StyleRegistry (/Users/fun117/_dev/github/selcold/sglid/website/node_modules/styled-jsx/dist/index/index.js:449:36)
    at eU (/Users/fun117/_dev/github/selcold/sglid/website/node_modules/next/dist/compiled/next-server/pages.runtime.dev.js:8:20468)
    at eH (/Users/fun117/_dev/github/selcold/sglid/website/node_modules/next/dist/compiled/next-server/pages.runtime.dev.js:17:1765)
    at eJ (/Users/fun117/_dev/github/selcold/sglid/website/node_modules/next/dist/compiled/next-server/pages.runtime.dev.js:17:3068)
    at div
    at e9 (/Users/fun117/_dev/github/selcold/sglid/website/node_modules/next/dist/compiled/next-server/pages.runtime.dev.js:26:761)   
```

**Comment by justincavery** [comment]
I've got the same issue, is there a suitable fix for it yet?

**Comment by Fun117** [comment]
> I've got the same issue, is there a suitable fix for it yet?

I don't know how to fix it either.

**Comment by dimaMachina** [comment]
please provide reproduction

**Comment by drakejwong** [comment]
Fixed this by upgrading Next from `14.1.3` to `14.2.14`. Seems to have been an issue with React ~`18.3.1`.

See [related discussion](https://github.com/vercel/next.js/issues/65161).
